common/utils.js

```diff
@@ -492,6 +503,10 @@
         }
 
 
+        // TODO: If we decide to catch errors in this file instead of call site,
+        // then we could wrap each of the returned functions with the catcher fn
+        // so that it's all in one place. Same with the other util functions returned
+        // in the factories/services in this file.
```

Comment:
[For Line 608](https://github.com/informatics-isi-edu/chaise/blob/rec-errors/common/utils.js#L608) we want to wrap both functions.